apps/json-viewer/package.json

```diff
@@ -48,6 +48,7 @@
     "@types/react": "18.0.9",
     "@types/react-dom": "18.0.3",
     "cross-env": "7.0.3",
+    "jsdom": "^25.0.1",
```

Comment:
What is the addition of this library for? Wondering if I need it for any of the apps I'm working on.

Reply:
its to make the tests pass